### ISSUE TITLE
Ignoring [Char] String type alias warning

### ISSUE THREAD

**Issue by lpil** [ndmitchell/hlint#94]
Hello!

My script contains the line

```
type Hand = [Char]
```

HLint doesn't like this- understandably it wants me to use `String`. I think it makes more sense to use `[Char]` on this occasion, so I would like to be able to disable this warning for this type.

I cannot figure out if there is a pragma that would allow me to do this- is it possible?

Thanks,
Louis
